modules/cluster-logging-uninstall.adoc

```diff
@@ -60,9 +60,33 @@ Do not delete the `openshift-operators-redhat` project if other global operators
 
 .. Confirm the deletion by typing `openshift-operators-redhat` in the dialog box and click *Delete*.
 
-. Optional: Remove any OpenShift Logging persistent volume claims (PVC):
+.Next steps
+
+Uninstalling OpenShift Logging does not remove the persistent volume claims (PVCs). To preserve or delete the PVCs, persistent volumes (PVs), and associated data, you must take further action.
+
+
+[WARNING]
+====
+Releasing or deleting PVCs can delete PVs and cause data loss.
+====
+
+. Evaluate whether to keep the PVCs for reuse with other pods. If so, keep the labels or PVC names you need to reclaim the PVCs.
+
+. Optional: If you do not want to reclaim the PVCs, you can delete them.
++
+If a PV's `persistentVolumeReclaimPolicy` is `delete`, deleting the PVC deletes its PV. This action releases the backing storage and removes all data, such as collected log data.
 
 .. Switch to the *Storage* -> *Persistent Volume Claims* page.
 
 .. Click the Options menu {kebab} next to each PVC and select *Delete Persistent Volume Claim*.
 
+.. If the PV's `persistentVolumeReclaimPolicy` is `retain`, use the `kubectl delete pv` command to delete the PVs. For example, enter:
```

Comment:
You could minimise this step by removing the command from the sentence. For example:

```
If the `persistentVolumeReclaimPolicy` parameter of a PV is set to `retain`, delete the PVs:
+
[source,terminal]
----
$ kubectl delete pv --all
----
```